tools/tdbg/flags.go

```diff
@@ -86,4 +86,9 @@ var (
 	FlagBase64File                 = "base64-file"
 	FlagTaskCategoryID             = "task-category-id"
 	FlagEncoding                   = "encoding"
+	FlagPartitionID                = "partition-id"
+	FlagStickyName                 = "sticky-name"
+	FlagBuildIDs                   = "build-ids"
+	FlagUnversioned                = "unversioned"
+	FlagAllActive                  = "all-active"
```

Comment:
to be consistent with the `temporal` CLI:
```suggestion
	FlagBuildIDs                   = "select-build-id"
	FlagUnversioned                = "select-unversioned"
	FlagAllActive                  = "select-all-active"
```